packages/vite/src/node/ssr/__tests__/ssrLoadModule.spec.ts

```diff
@@ -11,6 +11,9 @@ async function createDevServer() {
     configFile: false,
     root,
     logLevel: 'silent',
+    optimizeDeps: {
+      disabled: true,
+    },
```

Comment:
Was this needed for the PR?

Reply:
Not really. A lot of changes actually wasn't needed until node14 start messing up everything 😄 Only the first commit was the real changes.

But I think the changes are fine still including this one, since for this case the unit test won't spin up an unused deps optimizer instance (for this PR onwards). I'm happy to revert it too though and we can check separately.

Reply:
Nah, let's keep it :)